spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/jsbridge/SCJavascriptBridgeHandler.java

```diff
@@ -216,17 +200,51 @@ public void onNext(Boolean updated) {
                 e.printStackTrace();
               }
             }
+            if (command.equals(BridgeCommand.DATASERVICE_CREATEFEATURE)) {
+              try {
+                SCSpatialFeature newFeature = getNewFeature(bridgeMessage.get("payload").get("feature").asText());
+                newFeature.setStoreId(bridgeMessage.get("payload").get("storeId").asText());
+                newFeature.setLayerId("point_features");  // TODO: update spatialconnect-js to send layerId oncreateFeature
```

Comment:
update spatialconnect-js first b/c we don't want to hardcode anything into the sdk